system/assets/js/admin/presidecore/preside.uber.select.js

```diff
@@ -529,17 +531,22 @@
 			  , filterInput, filterByValue;
 
 			if ( filterBy !== null && filterBy.length ) {
-				filterInput = $( "input[name='" + filterBy + "']" );
+				filterBy      = filterBy.split( ',' );
+				filterByField = filterByField.split( ',' );
 
-				if ( filterInput.length ) {
-					this.filter_field = filterInput;
-					filterByValue = this.filter_field.val();
-				} else {
-					filterByValue = cfrequest[ filterBy ] || null;
-				}
+				for( var i=0; i<filterBy.length; i++ ) {
```

Comment:
Define `var` variables at the top of function definition with js (see Pixl8 styleguide)